extensions/micrometer/runtime/pom.xml

```diff
@@ -103,6 +103,7 @@
             <optional>true</optional>
         </dependency>
 
+
```

Comment:
Cosmetic - this new line could be removed.

Reply:
Oops

Reply:
Fixed